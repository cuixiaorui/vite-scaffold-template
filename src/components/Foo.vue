<template>
  <div>foo</div>
</template>

<script lang="ts"></script>

<style scoped></style>
