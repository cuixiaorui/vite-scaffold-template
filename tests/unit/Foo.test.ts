// import Foo from '../../src/components/Foo.vue'
import Foo from '@/components/Foo.vue'
import { render } from '@testing-library/vue'
it('snapshot', () => {
  const { container } = render(Foo)
  expect(container).toMatchSnapshot()
})
