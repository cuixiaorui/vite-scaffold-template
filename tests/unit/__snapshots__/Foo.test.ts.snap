// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<div>
  <div>
     foo ----------------- 
    <button>
      login
    </button>
    <button>
      get user
    </button>
    <div>
      bar
    </div>
  </div>
</div>
`;
